ttnn/cpp/ttnn/operations/eltwise/binary/binary_pybind.hpp

```diff
@@ -44,8 +44,8 @@ void bind_primitive_binary_operation(py::module& module, const binary_operation_
 
         Example:
 
-            >>> tensor1 = ttnn.to_device(ttnn.from_torch(torch.tensor((1, 2), dtype=torch.bfloat16)), device)
-            >>> tensor2 = ttnn.to_device(ttnn.from_torch(torch.tensor((0, 1), dtype=torch.bfloat16)), device)
+            >>> tensor1 = ttnn.to_device(ttnn.from_torch(torch.tensor(([[1, 2], [3, 4]]), dtype=torch.bfloat16)), device)
+            >>> tensor2 = ttnn.to_device(ttnn.from_torch(torch.tensor(([[1, 2], [3, 4]]), dtype=torch.bfloat16)), device)
             >>> output = {1}(tensor1, tensor2)
```

Comment:
What was the issue ? Is it zero or the format how it passed?

Reply:
The format that we passed.
`torch.tensor((1, 2), dtype=torch.bfloat16)` creates a tensor of rank 1 with elements [1  2].
According to the documentation, most of the ops support ranks of 2, 3, 4. That's why we passed the input as 
[1 2
3 4]